src/Fields/field.jl

```diff
@@ -859,3 +859,9 @@ function fill_halo_regions!(field::Field, positional_args...; kwargs...)
 
     return nothing
 end
+
+#####
+##### nodes
+#####
+
+nodes(f::Field; kwargs...) = nodes(f.grid, instantiated_location(f)...; indices=indices(f), kwargs...)
```

Comment:
```suggestion
nodes(f::Field; kwargs...) = nodes(f.grid, instantiated_location(f)...; indices=indices(f), kwargs...)

```

Reply:
I think I'm missing something; What's different here?

Reply:
the extra line at the bottom of the file